### PR TITLE
[improve][broker][PIP-149]make getMaxConsumersPerSubscription method async in Namespaces

### PR DESCRIPTION
**Pull request by HQebupt**
### Motivation

See #14365 

### Verifying this change

Make sure that the change passes the CI checks.

### Does this pull request potentially affect one of the following parts:

If `yes` was chosen, please highlight the changes

- Dependencies (does it add or upgrade a dependency): (no)
- The public API: (no)
- The schema: (no)
- The default values of configurations: (no)
- The wire protocol: (no)
- The rest endpoints: (no)
- The admin cli options: (no)
- Anything that affects deployment: (no)

### Documentation
Check the box below and label this PR (if you have committer privilege).

Need to update docs? 
- [x] `doc-not-needed`

### CODE REVIEW

**Review by AnonHxy — APPROVED** [review]

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java [hunk 1]
```diff
@@ -1325,11 +1325,21 @@ public void setMaxConsumersPerTopic(@PathParam("property") String property, @Pat
...
-    public Integer getMaxConsumersPerSubscription(@PathParam("property") String property,
-                                              @PathParam("cluster") String cluster,
-                                              @PathParam("namespace") String namespace) {
+    public void getMaxConsumersPerSubscription(
+            @Suspended final AsyncResponse asyncResponse,
+            @PathParam("property") String property,
+            @PathParam("cluster") String cluster,
+            @PathParam("namespace") String namespace) {
         validateNamespaceName(property, cluster, namespace);
-        return internalGetMaxConsumersPerSubscription();
```
Comment (codelipenghui) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java:1332]: Looks like we can remove `internalGetMaxConsumersPerSubscription`?

Comment (HQebupt) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java:1332]: Sure.

**Review by gaoran10 — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**